README.md

```diff
@@ -80,6 +81,14 @@ Let's test an Appdaemon automation we created, which, say, handles automatic lig
        given_that.state_of('sensor.living_room_illumination').is_set_to(200) # 200lm == night
        living_room._new_motion(None, None, None)
        assert_that('light.living_room').was.turned_on()
+       
+       # Change entity states 
+       given_that.state_of('device_tracker.person).is_set_to('away')
+       given_that.mock_functions_are_cleared()
+       
+       living_room._new_motion(None, None, None)
+       assert_that('light.living_room').was_not.turned_on()
+       
```

Comment:
Thanks for updating the README. That's really cool! 

Unfortunately, I have to disagree with this change. The reason is the fact it crunches 2 unit tests into one, and I'd like to keep the quick-start simple. Let me explain:

As you can see, you have to reset the mock functions in the middle `given_that.mock_functions_are_cleared()`. Why is that? ==> At line 85, a new unit test starts.
After the first `assert` at line 83, the test has been concluded, and a new test starts right below: It has it's own fixture (state is away), it's own action (another new motion triggered) and it's own assertion (was not turned on). If it were in its own separate test the functions would have been already cleared in the initialization fixture, thus eliminating the need for `given_that.mock_functions_are_cleared()`, narrowing down the case being tested, and overall improving readability/maintainability.

Now why not have 2 separate unit tests in the quick start guide? Because I'd like to keep it quick and simple. More in-depth example can be found in the `doc/full_example` folder. Feel free to add as many tests as you'd like there. Even create your own apps and tests to showcase corny scenario you encounter and want to keep track of. This would be very welcome :) Even better, you can do so in the `test/integration_tests` folder, any tests you implement there will be run as part of the continuous delivery mechanism. Ensuring whatever change is done will never break the way you understood the framework at the time you implemented the test ;)

Reply:
I definitely see how it would make sense as its own separate test :)

Regarding your question: Isn't it already shown at line 81? 
`given_that.state_of('sensor.living_room_illumination').is_set_to(200)`

(Sorry for the late response, during the holidays my availability is limited)

Reply:
You've got a good point! 👍

I'd like to keep the quick-start straightforward, but I'll open a PR to discuss where we could showcase this scenario.